pages/services/spinnaker/0.3.0-1.9.2/quick-start-guide/index.md

```diff
@@ -0,0 +1,381 @@
+    {
+      "gracePeriodSeconds": 120,
+      "intervalSeconds": 30,
+      "maxConsecutiveFailures": 3,
+      "timeoutSeconds": 20,
+      "delaySeconds": 15,
+      "protocol": "COMMAND",
+      "command": {
+        "value": "/usr/bin/healthcheck.sh"
+      }
+    }
+  ],
+  "instances": 1,
+  "maxLaunchDelaySeconds": 36000,
+  "mem": 1024,
+  "gpus": 0,
+  "networks": [
+    {
+      "mode": "container/bridge"
+    }
+  ],
+  "residency": {
+    "relaunchEscalationTimeoutSeconds": 3600,
+    "taskLostBehavior": "WAIT_FOREVER"
+  },
+  "requirePorts": false,
+  "upgradeStrategy": {
+    "maximumOverCapacity": 0,
+    "minimumHealthCapacity": 0
+  },
+  "killSelection": "YOUNGEST_FIRST",
+  "unreachableStrategy": "disabled",
+  "fetch": [],
+  "constraints": []
+}
+```
+
+and run the following two commands:
+
+```
+dcos package install marathon-lb --yes
+dcos marathon app add misc/minio.json
+```
+
+`marathon-lb` will make the minio console accessible via the DC/OS public agent. In your browser enter the following address. The `minio` credentials are minio / minio123.
+```
+http://<public-agent-ip>:9000
+```
+
+Out of the box the DC/OS `Spinnaker` service allows you to deliver to the `DC/OS cluster` the service runs itself in. The Spinnaker `deck` and `gate` services will be made available via a proxy or edge-lb running on the DC/OS clusters public agent. Note down the hostname/ip of the public agent in your DC/OS cluster
+
+
+With that we are ready to install `Spinnaker`. In the DC/OS catalog/universe select Spinnaker which will show you the following. Hit *Review&Run*.
+
+[<img src="/services/spinnaker/0.3.0-1.9.2/img/inst01.png" alt="Spinnaker Quick Start"/>](/services/spinnaker/0.3.0-1.9.2/img/inst01.png)
+
+In the service section fill in the proxy hostname with the hostname of the public agent noted down earlier.
+
+[<img src="/services/spinnaker/0.3.0-1.9.2/img/inst02.png" alt="Spinnaker Quick Start"/>](/services/spinnaker/0.3.0-1.9.2/img/inst02.png)
+
+If for `minio` and `DC/OS` the default credentials are use then you are ready to go hit *Review&Run*. Otherwise the next two steps show how to configure your specific credentials.
+
+[<img src="/services/spinnaker/0.3.0-1.9.2/img/inst03.png" alt="Spinnaker Quick Start"/>](/services/spinnaker/0.3.0-1.9.2/img/inst03.png)
+
+The following dialog shows you how to configure the `minio` credentials.
+
+[<img src="/services/spinnaker/0.3.0-1.9.2/img/inst04.png" alt="Spinnaker Quick Start"/>](/services/spinnaker/0.3.0-1.9.2/img/inst04.png)
+
+The following dialog shows you how to configure the `DC/OS` credentials.
+
+[<img src="/services/spinnaker/0.3.0-1.9.2/img/inst05.png" alt="Spinnaker Quick Start"/>](/services/spinnaker/0.3.0-1.9.2/img/inst05.png)
+
+
+Once the service is running we launch a simple `proxy` to get access to the Spinnaker `deck` and `gate` service. Use the following [proxy.json](misc/proxy.json). For how to use edge-lb to expose these services look [here]().
```

Comment:
add the proxy json here.

Reply:
also proxy should be in mesosphere DH account

Reply:
added proxy.json and also sample on how to use edge-b alternatively

and yes that image has to go to mesosphere site, along with the custom deck image